### PR TITLE
Add watch script

### PR DESCRIPTION
**Pull request by RomanLukash340**

### CODE REVIEW

File: build-watch.js
Comment (DavidSouther) [inline build-watch.js:1]: This build-watch file seems like overkill - the set of projects is well known, and simply listing each one should be fine.

1. Add a `watch` target in each package (components, projects, simulator) that does `npm run build -- --watch`
2. Add a `watch` target in the root, which calls those directly using concurrently: `conc \"npm run -w components watch\" \"npm run -w projects watch\" \"npm run -w simulator watch\"`

**Review by DavidSouther — CHANGES_REQUESTED** [review]

File: build-watch.js
Comment (happytomatoe) [reply build-watch.js:1]: Changed

File: package.json [hunk 1]
```diff
@@ -57,4 +56,4 @@
     "cross-env": "^7.0.3",
     "shx": "^0.3.4"
   }
-}
+}
```
Comment (DavidSouther) [inline package.json:59]: (nit) ending newline got lost

File: package.json [hunk 2]
```diff
@@ -28,8 +28,7 @@
     "prebuild": "npm run check",
     "build": "cross-env CI=true npm run build -w projects && cross-env CI=true npm run build -w runner && cross-env CI=true npm run build -w simulator && cross-env CI=true npm run build -w cli && cross-env CI=true npm run build -w components && cross-env CI=true npm run build -w web && cross-env CI=true npm run build -w extension",
     "build:extension": "cross-env CI=true npm run build -w extension",
-    "watch": "node build-watch.js",
-    
+    "watch": "concurrently -k --restart-tries 3 \"npm run build -w components -- -w \" \"npm run build -w projects -- -w\" \"npm run build -w runner -- -w\" \"npm run build -w simulator -- -w\"   ",
```
Comment (DavidSouther) [inline package.json:31]: Could you clean up the whitespace in here a bit? Sometimes there's extra spaces before and after the \", sometimes not.

Comment (happytomatoe) [reply package.json]: if you mean at then end - done

File: package.json (hunk 1 above)
Comment (happytomatoe) [reply package.json]: added newline

**Review by DavidSouther — APPROVED** [review]

**Pull request merged**